### PR TITLE
Test

### PR DESCRIPTION
**Pull request by pyncior**

### CODE REVIEW

**Review by anargu — COMMENTED** [review]
added a comment

File: src/pages/home/index.js [hunk 1]
```diff
@@ -14,7 +14,9 @@ import { Products } from "./Products";
 
 export const Home = (props) => {
   const router = useRouter();
+
```
Comment (sebastianmendo1995) [inline src/pages/home/index.js:17]: this is not correct

File: src/pages/home/index.js [hunk 2]
```diff
@@ -14,7 +14,9 @@ import { Products } from "./Products";
...
   const [authUser] = useGlobal("user");
+
   const [comments, setComments] = useState([]);
```
Comment (anargu) [inline src/pages/home/index.js:20]: added a comment in code

Comment (anargu) [reply src/pages/home/index.js:20]: @pyncior this is comment in code

**Review by anargu — APPROVED** [review]

**Pull request merged**